src/Symfony/Component/Security/Http/composer.json

```diff
@@ -19,7 +19,7 @@
         "php": "^7.1.3",
         "symfony/security-core": "^4.3",
         "symfony/http-foundation": "^3.4|^4.0|^5.0",
-        "symfony/http-kernel": "^4.3|^5.0",
+        "symfony/http-kernel": "^4.3",
```

Comment:
because of e.g. https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Http/Firewall/ExceptionListener.php#L90